index.html

```diff
@@ -1,5 +1,4 @@
----
```

Comment:
No, it isn't working yet :/
https://heredroky.github.io/gci16.fossasia.org/index.html